modules/cd-tekton-pipeline/src/main/java/com/ibm/cloud/continuous_delivery/cd_tekton_pipeline/v2/model/Property.java

```diff
@@ -45,175 +45,19 @@ public interface Type {
-      this.path = property.path;
-    }
-
-    /**
-     * Instantiates a new builder.
-     */
-    public Builder() {
-    }
-
-    /**
-     * Instantiates a new builder with required properties.
-     *
-     * @param name the name
-     * @param type the type
-     */
-    public Builder(String name, String type) {
-      this.name = name;
-      this.type = type;
-    }
-
-    /**
-     * Builds a Property.
-     *
-     * @return the new Property instance
-     */
-    public Property build() {
-      return new Property(this);
-    }
-
-    /**
-     * Adds an xEnum to xEnum.
-     *
-     * @param xEnum the new xEnum
-     * @return the Property builder
-     */
-    public Builder addXEnum(String xEnum) {
-      com.ibm.cloud.sdk.core.util.Validator.notNull(xEnum,
-        "xEnum cannot be null");
-      if (this.xEnum == null) {
-        this.xEnum = new ArrayList<String>();
-      }
-      this.xEnum.add(xEnum);
-      return this;
-    }
-
-    /**
-     * Set the name.
-     *
-     * @param name the name
-     * @return the Property builder
-     */
-    public Builder name(String name) {
-      this.name = name;
-      return this;
-    }
-
-    /**
-     * Set the value.
-     *
-     * @param value the value
-     * @return the Property builder
-     */
-    public Builder value(String value) {
-      this.value = value;
-      return this;
-    }
-
-    /**
-     * Set the href.
-     *
-     * @param href the href
-     * @return the Property builder
-     */
-    public Builder href(String href) {
-      this.href = href;
-      return this;
-    }
-
-    /**
-     * Set the xEnum.
-     * Existing xEnum will be replaced.
-     *
-     * @param xEnum the xEnum
-     * @return the Property builder
-     */
-    public Builder xEnum(List<String> xEnum) {
-      this.xEnum = xEnum;
-      return this;
-    }
-
-    /**
-     * Set the type.
-     *
-     * @param type the type
-     * @return the Property builder
-     */
-    public Builder type(String type) {
-      this.type = type;
-      return this;
-    }
-
-    /**
-     * Set the path.
-     *
-     * @param path the path
-     * @return the Property builder
-     */
-    public Builder path(String path) {
-      this.path = path;
-      return this;
-    }
-  }
-
   protected Property() { }
 
-  protected Property(Builder builder) {
-    com.ibm.cloud.sdk.core.util.Validator.notNull(builder.name,
-      "name cannot be null");
-    com.ibm.cloud.sdk.core.util.Validator.notNull(builder.type,
-      "type cannot be null");
-    name = builder.name;
-    value = builder.value;
-    href = builder.href;
-    xEnum = builder.xEnum;
-    type = builder.type;
-    path = builder.path;
-  }
-
-  /**
-   * New builder.
-   *
-   * @return a Property builder
-   */
-  public Builder newBuilder() {
-    return new Builder(this);
-  }
-
   /**
    * Gets the name.
    *
    * Property name.
    *
    * @return the name
    */
-  public String name() {
+  public String getName() {
```

Comment:
Wouldn't something like this would count as a breaking change? The same comment applies to the changes in this file

Reply:
Hmm yeah, I can rework things to keep them non-breaking, as these changes came from using a newer version of the SDK generator

Reply:
Fixed